packages/python-packages/api-stub-generator/tests/docstring_parser_test.py

```diff
@@ -0,0 +1,111 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from apistub.nodes import DocstringParser
+from apistub.nodes import ArgType
+
+docstring_standard_return_type = """
+Dummy docstring to verify standard return types and param types
+:rtype: str
+"""
+
+docstring_Union_return_type1 = """
+Dummy docstring to verify standard return types and param types
+:rtype: Union[str, int]
+"""
+
+docstring_Union_return_type2 = """
+Dummy docstring to verify standard return types and param types
+:rtype: Union(str, int)
+Dummy string at new line
+"""
+
+docstring_union_return_type3 = """
+Dummy docstring to verify standard return types and param types
+:rtype: union[str, int]
+"""
+
+docstring_multi_ret_type = """
+Dummy docstring to verify standard return types and param types
+:rtype: str or ~azure.test.testclass or None
+"""
+
+docstring_dict_ret_type = """
+Dummy docstring to verify standard return types and param types
+:rtype: dict[str, int]
+"""
+
+docstring_param_type = """
+:param str name: Dummy name param
+:param val: Value type
+:type val: str
+"""
+
+docstring_param_type1 = """
+:param str name: Dummy name param
+:param val: Value type
+:type val: str
+:param ~azure.core.pipeline pipeline: dummy pipeline param
+:param pipe_id: pipeline id
+:type pipe_id: Union[str, int]
+:param data: Dummy data
+:type data: str or 
+~azure.dummy.datastream
+"""
+
+class TestDocStringParser:
+
+    def _test_return_type(self, docstring, expected):
+        docstring_parser = DocstringParser(docstring)
+        print(docstring_parser.find_return_type())
```

Comment:
Normally tests don't output stuff aside from the pytest output. (my opinion at least)

Reply:
Oh, I have forgot to remove that line. will do it.